### PR TITLE
Add test for null String

### PR DESCRIPTION
**Pull request by nonguyen1**
Add a simple test for java `String` is `null` from Rust None `Option`.

### CODE REVIEW

File: test-crates/duchess-java-tests/tests/ui/passing_null_values.rs [hunk 1]
```diff
@@ -5,6 +5,13 @@ duchess::java_package! {
     package take_null;
 
     public class TakeNull { * }
+    public class TakeNullRecord { * }
```
Comment (rcoh) [inline test-crates/duchess-java-tests/tests/ui/passing_null_values.rs:8]: we need to change this test to `//@run` — currently it's not actually being run, just compiled.

**Review by rcoh — APPROVED** [review]
can you change the test to `//@run`?

**Comment by nonguyen1** [comment]
We'll need to hold off on this. After I updated the test to `//@run` I'm seeing some runtime failures. I'll try to take a look soon.

```rust
    let java = take_null_record.to_java().assert_not_null().execute()?;
    let is_null = java.is_null().execute().unwrap(); // <- `Error: attempted to deref a null Java object pointer`
    assert!(!is_null);
```

**Comment by nikomatsakis** [comment]
So is this fixed by https://github.com/duchess-rs/duchess/pull/182 ?

**Comment by nonguyen1** [comment]
Yup fixed by https://github.com/duchess-rs/duchess/pull/182
